app/notes/[noteId].tsx

```diff
@@ -78,8 +77,8 @@ export default function Note() {
   return (
     <View style={styles.container}>
       <View style={styles.textContainer}>
-        <Text style={styles.title}>{note.title}</Text>
-        <Text style={styles.textContent}>{note.textContent}</Text>
+        <Text style={styles.title}>{note?.title}</Text>
+        <Text style={styles.textContent}>{note?.textContent}</Text>
```

Comment:
this change doesn't look right - we shouldn't introduce optional chaining here

Reply:
36dde012dafc577628274c2899510c90f09456a7